llvm/test/ThinLTO/X86/Inputs/calloc.ll

```diff
@@ -0,0 +1,21 @@
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
+
+@_ZZ6mallocE3buf = internal global [2000 x i8] zeroinitializer, align 1
+
+define hidden ptr @malloc(i64 %a) #0 {
+entry:
+  ret ptr @_ZZ6mallocE3buf
+}
+
+define hidden ptr @calloc(i64 noundef %num, i64 noundef %size) {
+entry:
```

Comment:
I will further cut down this test case since the implementation of `calloc` doesn't matter here.